src/askui/agent.py

```diff
@@ -88,13 +88,36 @@ def click(self, instruction: Optional[str] = None, button: Literal['left', 'midd
             self.report.add_message("User", msg)
         if instruction is not None:
             logger.debug("VisionAgent received instruction to click '%s'", instruction)
```

Comment:
Should we not make this similar to mouse moving adding a message to the report á la "User click"?